department-of-veterans-affairs/caseflow
# Update "Review Tasks & Appeals" Step Continue Button Logic

Resolves [Intake Form Step 2- Continue Button](https://jira.devops.va.gov/browse/APPEALS-34717)

# Description
When a user indicates that there are tasks related to an existing appeal, this change now requires at least one appeal to be selected before the user can continue.

## Acceptance Criteria

## Testing Plan
1. First, run the following command to ensure the correspondence.rb file has seeded the relevant data for testing: `bundle exec rake db:seed:correspondence`
1. In rails console pull the uuid for the .last created correspondence
1. Open browser and got to http://localhost:3000/queue/correspondence/:correspondence_uuid/intake (in the link, use the UUID found in the rails console for previous step).
1. Navigate to the "Review Tasks & Appeals" step
1. Verify ticket behavior when selecting that there is/is not a related appeal.

# Frontend
## User Facing Changes

<img width="1375" alt="Screenshot 2023-11-16 at 15 41 00" src="https://github.com/department-of-veterans-affairs/caseflow/assets/2421172/3c4a0286-046c-4c49-b9e5-909e91792c36">
